src/NUnitFramework/mock-assembly/mock-assembly-netstandard13.csproj

```diff
@@ -63,14 +64,17 @@
     <None Include="..\..\nunit.snk">
       <Link>nunit.snk</Link>
     </None>
-    <None Include="mock-assembly-portable.project.json" />
+    <None Include="mock-assembly-netstandard13.project.json" />
   </ItemGroup>
   <Import Project="$(MSBuildExtensionsPath32)\Microsoft\Portable\$(TargetFrameworkVersion)\Microsoft.Portable.CSharp.targets" />
```

Comment:
Ah ok - I assumed that was Portable as in PCL, rather than Portable as in X-Plat. Sorry!